CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ endforeach()
 
 file(GLOB_RECURSE SCHEMAS LIST_DIRECTORIES FALSE RELATIVE "${CMAKE_SOURCE_DIR}/templates" "${CMAKE_SOURCE_DIR}/templates/*")
 foreach(ITEM IN ITEMS ${SCHEMAS})
-  if(NOT ITEM MATCHES "^include/" )
+  if(NOT ITEM MATCHES "^(include/|.*/\\.[^/])" )
```

Comment:
Why is this change necessary? There shouldn't be any hidden files. Is this for local development where there might be editor temp files or something?

Reply:
Exactly - for example I usually run tests while editing the files in another tab, which fails without this.

Reply:
Wouldn't you want to have saved your edited file *before* running the test? That aside, this does not seem to me to be an unreasonable change to make, so LGTM.

Reply:
Well yeah, but I can save while still having them open :) saves me an extra `nvim` cmd

Reply:
I'll chalk this up as an emacs vs. vim thing, since the scenario you describe makes no sense to me ;)